secrethub/resource_secret_test.go

```diff
@@ -106,6 +108,29 @@ func TestAccResourceSecret_generate(t *testing.T) {
 		}
 	`, testAcc.secretName, testAcc.path)
 
+	configSpecificCharsets := fmt.Sprintf(`
+		resource "secrethub_secret" "%v" {
+			path = "%v"
+			generate {
+				length = 16
+				charsets = ["numbers", "symbols"]
+			}
+		}
+	`, testAcc.secretName, testAcc.path)
+
+	configOneMinRule := fmt.Sprintf(`
+		resource "secrethub_secret" "%v" {
+			path = "%v"
+			generate {
+				length = 16
+				charsets = ["numbers", "letters"]
+				min = {
+					numbers = 15
+				}
+			}
+		}
+	`, testAcc.secretName, testAcc.path)
+
```

Comment:
Maybe add one testcase where we have a `min` but no `charsets` configured. That should be possible when the `min` charset is part of the alphanumeric charset, right? (e.g. `numbers`)

Reply:
Yes, and I'll also add some bad weather test cases.